Ball of Duty Server/Domain/Broker.cs

```diff
@@ -117,11 +207,19 @@ public void Receive()
                     }
                 });
             }
+            catch (Exception e)
```

Comment:
then the solution is to have two catches... otherwise you will silence every other exception